linker/Mono.Linker/Driver.cs

```diff
@@ -301,6 +304,7 @@ static void Usage (string msg)
 			Console.WriteLine ("   -b          Generate debug symbols for each linked module (true or false)");
 			Console.WriteLine ("   -g          Generate a new unique guid for each linked module (true or false)");
 			Console.WriteLine ("   -v          Keep memebers needed by debugger attributes (true or false)");
+			Console.WriteLine("    -n          Ignore unresolved types and methods (true or false)");
```

Comment:
Looks good but for such special cases we should do advanced option e.g. `--skip-unresolved` only

Reply:
Sure, I can change that but wouldn't it look odd that this will be the only such option (not counting --version and --about)?

Reply:
Don't worry the more will follow in the future